pkg/reconciler/example/route.go

```diff
@@ -0,0 +1,419 @@
+package main
```

Comment:
Look again, it’s looking up the ”actual” route from the route table maintained by the devices controller (which pulls routes over netlink). This way it can efficiently reconcile a lot of routes without having to do individual syscalls to fetch routes one by one.